index.js

```diff
@@ -229,6 +229,29 @@ const ContainerService = {
     return userContextId in this._identitiesState;
   },
 
+  _closeTabs(tabsToClose) {
+    // We create a new tab only if the current operation closes all the
+    // existing ones.
+    if (tabs.length !== tabsToClose.length) {
+      return Promise.resolve(null);
+    }
+
+    const browserWin = windowUtils.getMostRecentBrowserWindow();
+
+    // This should not really happen.
+    if (!browserWin || !browserWin.gBrowser) {
+      return Promise.resolve(null);
+    }
```

Comment:
This check is duplicated in `openTab` - can it be turned into a helper function?